README.md

```diff
@@ -294,6 +294,10 @@ Field overrides allow you to define custom GraphQL types or resolvers on top of
 
 for more information in Overrides, they adhere to the GraphQLFieldConfig outlined here https://graphql-compose.github.io/docs/basics/what-is-resolver.html
 
+## fieldTransform
+
+fieldTransform is a function that can be provided to the config to transform the names of fields before the schema is generated -- this is used internally to remove spaces but can be used for other global transforms as well
+
```

Comment:
[nit] this should be moved to the `@flatbread/config` README

Reply:
Well I could see an argument for surfacing it on the front page of the repo until we have a proper docs site.

How about:

```suggestion
## Advanced Config
### `fieldNameTransform`
Accepts a function which takes in field names and transforms them for the GraphQL schema generation -- this is used internally to remove spaces but can be used for other global transforms as well

// some example here

```